src/view/explorer/ExplorerPane.ts

```diff
@@ -203,6 +205,11 @@ export class ExplorerPane extends ItemView {
           item.setIcon("document");
           item.onClick(() => addRelativeScene("after", file));
         });
+        menu.addItem((item) => {
+          item.setTitle("Ignore scene");
```

Comment:
Technically the file is not meant to be a scene, so I would instead call this

```suggestion
          item.setTitle("Ignore note in Longform");
```